scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -42,8 +42,18 @@ fi
 
 WORKSPACE=/root/project
 
-echo -en 'travis_fold:start:compiling_solidity\\r'
 cd $WORKSPACE
+
+# shellcheck disable=SC2166
+if [[ "$CIRCLE_BRANCH" == release -o -n "$CIRCLE_TAG" -o -n "$FORCE_RELEASE" -o "$(git tag --points-at HEAD 2>/dev/null)" == v* ]]
```

Comment:
Will this work with these `CIRCLE` vars?

Reply:
Yes, on circleci. See also https://github.com/ethereum/solidity/pull/10072